admin_manual/ai/app_api_and_external_apps.rst

```diff
@@ -0,0 +1,23 @@
+========================
+AppAPI and External Apps
+========================
+
+.. _ai-app_api:
+
+Previously, Nextcloud only supported applications written in the PHP programming language. In order to support a wider range of use cases,
+the External App ecosystem was introduced, which allows installing apps that are docker containers.
+
+Most of our AI apps nowadays are ExApps (External Apps). And thus require some preparation of your Nextcloud before you can install them.
+
+Installation
+------------
+
+1. You will need to install the `AppAPI <https://apps.nextcloud.com/apps/app_api>`_ app from the app store.
+2. You will need to set up a Deploy Daemon in the AppAPI admin settings. A Deploy Daemon is an endpoint that allows the Nextcloud instance to install and communicate with External Apps.
+
+   1. You will need to setup a docker container called *Docker-Socket proxy* as outlined in the following link: https://github.com/cloud-py-api/docker-socket-proxy#readme
```

Comment:
> A Deploy Daemon is an endpoint that allows the Nextcloud instance to install and communicate with External Apps.

It is more like a deployment and interface configuration for External Apps imo.

> You will need to setup a docker container called *Docker-Socket proxy*

"You will need to setup a docker socket proxy for AppAPI to securely access the docker socket. The setup process is outlined in the following link: https://github.com/cloud-py-api/docker-socket-proxy#readme"